src/sentry/static/sentry/app/components/userBadge.jsx

```diff
@@ -3,17 +3,18 @@ import React from 'react';
 import styled from 'react-emotion';
 import Avatar from './avatar';
 import Link from './link';
+import overflowEllipsis from '../styles/overflowEllipsis';
 
-const UserBadge = ({user, orgId}) => {
+const UserBadge = ({user, orgId, ...props}) => {
   return (
-    <StyledUserBadge>
+    <StyledUserBadge {...props}>
       <StyledAvatar user={user} size={80} className="avatar" />
-      <div>
+      <StyledNameAndEmail>
```

Comment:
This is just a discussion comment, no changes needed here, but we should think about how to name these styled components. There's two basic cases that we see a lot:

### Extending a DOM el (e.g. `div`)
Because we're styling a native DOM el, there's no semantic name provided (vs a component) and I think in this case we should avoid `Styled`-prefix because it adds a bit of noise. This also helps when we're writing tests because we can search via component's display name.

### Extending components
Naming things is hard and the component name already gives us some context, so I think `Styled` prefix works fine here.

Thoughts?

Reply:
Yeah that seems right. 

It might be nice to have some kind of indicator that an element is strictly presentational and not a real component: when I come in cold to a new template, it can be frustrating trying to figure out what is a styled div and what is an actual component, but we can also just figure that out later.

Reply:
Yeah I was thinking about that too, don't really have an answer yet.

They're still "real" components and their interface is no different than a non-presentational component.

Reply:
Here is kind of a radical proposal: when applying presentational styles to a `div` or `a` we could use `css` and `classNames` instead:

```js
const nameAndEmail = ({user}) => 
  <div className={nameAndEmailStyles}>
    <StyledAvatar user={user} />
    {user.email} | {user.name}
  </div>
);

const nameAndEmailStyles = css`
  display: flex;
  align-items: center;
`;

export default nameAndEmail;
```

That way we can avoid creating components when we don't need to, and popular style blocks could be imported like utility classes

```js

import {verticalMarginsLarge, horizontalMarginsSmall} from './helpers/margins';

const Paragraph = ({children}) => (
  <p className={cx(verticalMarginsLarge, horizontalMarginsSmall)}>
    {children}
  </p>
);

```